kyo-core/shared/src/test/scala/kyoTest/internal/PositionTest.scala

```diff
@@ -24,7 +24,7 @@ class PositionTest extends KyoTest:
         assert(e == "PositionTest.scala:21 - ")
     }
     "WithOwner" - {
-        import Position.WithOwner.given Position
+        inline given Position = Position.WithOwner.derive
```

Comment:
Unrelated compilation warning fix. The compiler was reporting the implicit as not used.

Reply:
Thanks